pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -309,6 +317,7 @@ func (c *Cloud) ensureLoadBalancerv2(namespacedName types.NamespacedName, loadBa
 		}
 
 		// Subnets cannot be modified on NLBs
+		// TODO: Add an error if user attempts to modify subnetmappings explicitly
```

Comment:
Q. Could you help me understand this comment? I see only describe operation here. Subnet mapping is only updated lines above. When would we modify subnet mapping in this?

Reply:
I may have been being too user friendly, but attempting to modify subnet mappings *including* changing the EIP allocation is not allowed on NLBs. If the user changes the annotation for EIP only then the EIP on the NLB would not change. That may be unexpected or counter-intuitive to a new user or one not familiar with AWS NLB. I keyed in on the comment regarding modifying subnet mappings and piggy backed my comment. I can remove the comment if you think it's confusing. It was more a marker for me to review how and when NLB changes get propagated.

Reply:
Fixed in f4a571f102